charts/hivemq-operator/values.yaml

```diff
@@ -2,7 +2,8 @@ global:
   rbac:
     create: true
     # Create a PodSecurityPolicy, cluster role, role binding and service account for the HiveMQ pods and assign the service account to them.
```

Comment:
Are the other parts of the comment not valid any more as well, role binding and service account and so on...